Revise technical support guide for Partner Center (#4043)
Microsoft/windows-driver-docs: windows-driver-docs-pr/dashboard/technical-support.md

- **Open a case directly with program support via Engage Center** (You must be signed in with a Partner Center account that is enrolled in the Hardware Program → use [Microsoft Engage Center](https://engagecenter.microsoft.com/#view/Microsoft_AzureCXP_EngageHub/EngageHubMenu.MenuView/targetExperience/sfb/sapId/bc9d4067-7218-61b9-1d2c-68ae591acf9d) *(fastest resolution)*.
- **You can't sign in with an account that has completed enrollment in the Hardware Program** (for example, your enrollment is incomplete, your account is suspended, or you can't access the enrolled tenant) → use [Partner Center support](https://partner.microsoft.com/support?stage=2&topicid=abf19e05-3f3a-f111-88b3-000d3a16941b).

## Open a case in Microsoft Engage Center (recommended)

Microsoft Engage Center is the fastest path to reach the Hardware Program support team.

1. Go to the Hardware Program support creation form in [Microsoft Engage Center](https://engagecenter.microsoft.com/#view/Microsoft_AzureCXP_EngageHub/EngageHubMenu.MenuView/targetExperience/sfb/sapId/bc9d4067-7218-61b9-1d2c-68ae591acf9d)

2. Sign in with your **Partner Center account that is enrolled in the Hardware Program**.

   > [NOTE]

>    > You must sign in with an account that is enrolled in the Hardware Program. If you sign in with a different account, you will be blocked selecting the correct routing choices during the creation workflow. If you can't sign in with an enrolled account, follow the steps in below instead.

Reviewer: ```suggestion
   > You must sign in with an account that is enrolled in the Hardware Program. If you sign in with a different account, you are blocked from selecting the correct routing choices during the creation workflow. If you can't sign in with an enrolled account, use the following steps instead.
```

3. When creating the case, select the following values:

   | Field | Value |
   |---|---|
   | Product | **Windows Developer Center** |
   | Problem Type | **Hardware Device Drivers** |

4. Complete the form and submit the case. A confirmation email with a support request number will be sent to the contacts confirming receipt of the submission.

## Can't sign in with an enrolled account? Open a Partner Center enrollment case

If you can't sign in to Engage Center with an account that is enrolled in the Hardware Program — for example, your enrollment is incomplete, you can't access the enrolled tenant, or you're trying to enroll for the first time — open a case through Partner Center support instead.

1. Go to [Partner Center support](https://partner.microsoft.com/support?stage=2&topicid=abf19e05-3f3a-f111-88b3-000d3a16941b).

2. Under **Next Step** choose the option to 'Provide issue details'

> 3. Complete the form and submit the case. A confirmation email with a support request number will be sent to the contacts confirming receipt of the submission.

Reviewer: ```suggestion
2. Under **Next Step**, choose the option **Provide issue details**

3. Complete the form and submit the case. A confirmation email with a support request number is sent to the contacts.
```
